app/views/contexts/section_summary_context.py

```diff
@@ -21,10 +24,20 @@ def __call__(self, current_location, return_to="section-summary"):
             if isinstance(title_for_location, dict)
             else title_for_location
         )
+
+        page_title = self._schema.get_custom_page_title_for_section(
```

Comment:
Minor, but the logic would be a little clearer if `page_title` was renamed to `custom_page_title`.

Reply:
Done in 79a79893ab71